### PR TITLE
fixes Comlaude

### PR DESCRIPTION
**Pull request by yassine-ouaamou**
Pull request by customer

### Proposed changes

* Tests conducted and normally the loop no longer exists.
* Regarding the relationships between indicators and observables, the indicators are now correctly linked to the observables.
* Put "Comlaude" as author.
* Added TLP Amber as "marking".
* Deleted useless label.
* The connector no longer re-imports the observables and indicators after it has finished importing the observables and indicators.

### Related issues

*
*

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality using different use cases
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: external-import/comlaude/src/main.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
 
 import yaml
 from pycti import OpenCTIConnectorHelper, get_config_variable
-from stix2 import Bundle, DomainName, Indicator, Relationship
+from stix2 import Bundle, DomainName, Indicator, Relationship, Identity
+from stix2.v21 import TLP_AMBER
```
Comment (Megafredo) [inline external-import/comlaude/src/main.py]: You don't need to specify the stix2.v21 import for TLPs:
Example :
either “stix2.TLP_AMBER” is enough
either in the import "from stix2 import TLP_AMBER"

File: external-import/comlaude/src/main.py [hunk 2]
```diff
@@ -87,6 +88,12 @@ def _generate_dynamic_custom_properties(helper, domain_object, score):
     return domain_name, custom_properties
 
 
+AUTHOR_NAME = "Comlaude"
```
Comment (Megafredo) [inline external-import/comlaude/src/main.py]: In the config.yml we can re-use the connector name, by adding a get_config_variable:
self.connector_name = get_config_variable(
 "CONNECTOR_NAME", ["connector", "name"], self.config,
)

File: external-import/comlaude/src/main.py [hunk 3]
```diff
@@ -87,6 +88,12 @@ def _generate_dynamic_custom_properties(helper, domain_object, score):
...
+
+
+author_identity = Identity(name=AUTHOR_NAME, identity_class="organization")
```
Comment (Megafredo) [inline external-import/comlaude/src/main.py]: If possible, add the generate_id to have predictive ids and limit certain errors
from pycti import Identity
...
stix2.Identity(  
    id=Identity.generate_id(name, "organization"),                // pycti -> Identity  
    name=name, 
    identity_class="organization", 
)

File: external-import/comlaude/src/main.py [hunk 4]
```diff
@@ -285,16 +305,13 @@ def run(self):
             self.comlaude_search.get_next_page()
             self._iterate_events()
 
-        if self.helper.connect_run_and_terminate:
-            self.helper.log_info(
-                "Connector stop: ({})".format(
-                    _format_time(datetime.datetime.now(datetime.UTC))
-                )
+        self.helper.log_info(
```
Comment (Megafredo) [inline external-import/comlaude/src/main.py]: Just for information! The new way to log is in this format: 
Ex: self.helper.connector_logger.info("your message", {"foo": foo, "bar": bar})

File: external-import/comlaude/src/main.py [hunk 5]
```diff
@@ -285,16 +305,13 @@ def run(self):
             self.comlaude_search.get_next_page()
             self._iterate_events()
 
-        if self.helper.connect_run_and_terminate:
```
Comment (Megafredo) [inline external-import/comlaude/src/main.py:288]: Why did you remove the run_and_terminate condition ?

File: external-import/comlaude/src/main.py [hunk 6]
```diff
@@ -285,16 +305,13 @@ def run(self):
...
-                    _format_time(datetime.datetime.now(datetime.UTC))
-                )
+        self.helper.log_info(
+            "Connector stop: ({})".format(
+                _format_time(datetime.datetime.now(datetime.UTC))
             )
-            self.helper.force_ping()
-            sys.exit(0)
-        # Sleep for interval specified in Hours.
-        time.sleep(self._get_interval())
```
Comment (Megafredo) [inline external-import/comlaude/src/main.py:297]: Why did you remove the interval ?

File: external-import/comlaude/src/main.py (hunk 5 above)
Comment (MohamedMerimi) [reply external-import/comlaude/src/main.py:288]: Hello Megradredo, i deleted the run and terminate condition because with this the script did a loop all the time

File: external-import/comlaude/src/main.py (hunk 1 above)
Comment (MohamedMerimi) [reply external-import/comlaude/src/main.py]: Hello Megrafredo, ok it's noted, i will modified the script

File: external-import/comlaude/src/main.py (hunk 3 above)
Comment (MohamedMerimi) [reply external-import/comlaude/src/main.py]: Hello Magafredo, Yes i think it's possible i will add the generate_id to have predictive ids

File: external-import/comlaude/src/main.py (hunk 4 above)
Comment (MohamedMerimi) [reply external-import/comlaude/src/main.py]: Hello, ok it's noted thanks for information

File: external-import/comlaude/src/main.py (hunk 6 above)
Comment (MohamedMerimi) [reply external-import/comlaude/src/main.py:297]: Hello, yes was my misstake sorry, i will add the interval
